Makefile

```diff
@@ -32,9 +32,9 @@ TARGETS := \
 # Build all ELF binaries using a Dockerized LLVM toolchain.
 docker-all:
 	docker run --rm --user "${UIDGID}" \
-		-v "${REPODIR}":/ebpf -w /ebpf \
+		-v "${REPODIR}":/ebpf -w /ebpf --env MAKEFLAGS \
 		"${IMAGE}:${VERSION}" \
-		make all
+		$(MAKE) all
```

Comment:
Path to `make` on the host != path to `make` in the container, so I would probably leave this as is. Nice addition of `MAKEFLAGS` though, was thinking we should probably pass that down into the container. :+1: